lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -376,7 +395,7 @@ def assign(src, dst)
     end
 
     def assign_later(parent:, child:)
-      buffers[:assignments] << [parent, child]
+      buffers[:assignments].merge!([parent.unique_identifier, child.unique_identifier] => [parent, child])
```

Comment:
for the record, we can't use `id` in the create case because the `id`s may or may not exist at the time we insert them into the buffer, depending on whether or not the model has been persisted.